packages/fetch-retry/package.json

```diff
@@ -8,8 +8,7 @@
   ],
   "scripts": {
     "test": "jest test",
-    "lint": "cross-env TIMING=1 eslint *.js",
-    "prettier": "prettier --write \"*.js\""
+    "lint": "cd ../.. && cross-env TIMING=1 eslint --config .eslintrc.js packages/fetch-retry/**/*.js"
```

Comment:
I think as @jaredpalmer said, you could run this without the `cd ../..` part, as it would go up the tree and find the config in root.

This also won't catch TypeScript files. In other repos, we just run this command.
```
eslint .
```

You might also consider adding:
```
--max-warnings=0
```